src/openseadragon.js

```diff
@@ -1248,7 +1256,7 @@ function OpenSeadragon( options ){
             },
             zoomPerClick:           2,
             zoomPerScroll:          1.2,
-            zoomPerDblTapDrag:      1.2,
+            zoomPerDblClickDrag:      1.01,
```

Comment:
I think it would be better to keep this number at 1.2 and reinstate the `/ 50` in the zoom equation. That presents a more consistent API to the developer.